### DISCUSSION TITLE
Class Arr is not found

### DISCUSSION THREAD

**Discussion by eduardonwa** [pboivin/filament-peek#141] [Q&A]
I looked everywhere in my project to see where I was using this class and I couldn't find any class or view where I was missing an Arr import. I don't really remember if I changed any line that indirectly had to do with the error.

So I went to ```vendor / pboivin / filament-peek / resources / views / preview-modal.blade.php``` and put these lines at the top:

```
@php
use Illuminate\Support\Arr;
@endphp
```

This peek file seems to be the only place where it is used and my problem disappeared once i imported it there. I don't think it's the best way to solve this but for now this fixes it, my question is how to really solve this?

The error I get is simply a class that is not found and it asks me to import it but as I mentioned before I can't find a file where Arr is being used. I would really appreciate your answers.

I am using versions : * v2.2.10

**Comment by pboivin** [comment]
The `Arr` alias is normally available by default to all Blade views. Did you change anything in the `alias` section of your `config/app.php`?

I guess it wouldn't hurt to use the full class name in Peek views. Feel free to send a PR if you have some time.

More info:

https://github.com/laravel/framework/blob/11.x/config/app.php#L186

**Comment by pboivin** [comment]
Fixed in https://github.com/pboivin/filament-peek/releases/tag/v2.2.11
